css-three-column-layout/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div id="container">
+
+    <div>
+      <h2 class="text">
+        Designed for life science. optimized for SaMD and Digital Health.
+      </h2>
+    </div>
+
+    <div class="box">
+
+      <div class="row">
+        <section class="col" id="col-1">
```

Comment:
This wrapper isn't actually providing any functionality based off of my note above ^. Since your parent `div` above it is technically acting as a column this isn't actually doing anything.